Automattic/pocket-casts-ios
# Search Improvements: add "Search History" view

| 📘 Project: #709 |
|:---:|

Adds the "Search history" view. This view doesn't have any function and all the data displayed is mocked.

| Light mode | Dark Mode |
| ----------- | ----------- |
| ![Simulator Screen Shot - iPhone 14 - 2023-02-21 at 14 22 10](https://user-images.githubusercontent.com/7040243/220416398-4a65e76c-7470-45c8-8d3f-06b379dda47b.png) | ![Simulator Screen Shot - iPhone 14 - 2023-02-21 at 14 21 49](https://user-images.githubusercontent.com/7040243/220416426-482662e1-136b-4d37-96b6-d36000fe4fc4.png) |

## To test

1. Run the app
1. In Settings > Beta Feature > enable `newSearch`
1. Under Podcasts or Discover tap the search
1. ✅ The search history should appear

You can test multiple themes and different font sizes in the Swift UI preview.

### Additional information

You'll notice that separators are not quite the same as the Figma designs. This is due to some limitations on iOS 14. Once we reach an agreement on whether to stop supporting it or not I'll make the appropriate changes.

## Checklist
